client/controller/index.web.js

```diff
@@ -4,36 +4,52 @@
 import React from 'react';
```

Comment:
It's because we need separate client/server versions of the controller, in combination with the `package.json` `browser` field trick.